rocky/katalogus/views/plugin_enable_disable.py

```diff
@@ -37,9 +37,7 @@ def post(self, request, *args, **kwargs):
             messages.add_message(
                 self.request,
                 messages.ERROR,
-                _("Failed fetching settings for boefje {boefje_name}. Is the Katalogus up?").format(
-                    boefje_name=self.plugin.name
-                ),
+                _("Failed fetching settings for {}. Is the Katalogus up?").format(self.plugin.name),
```

Comment:
We should also use `self.plugin.type.title()` in the message here and the INFO message below just like the other two messages.